src/Articles.jsx

```diff
@@ -0,0 +1,24 @@
+import { useState, useEffect } from "react";
+import ArticleCard from "./ArticleCard";
+
+export default function Articles() {
+  const [articlesList, setArticlesList] = useState([]);
+
+  useEffect(() => {
+    fetch("https://rafaels-nc-news.onrender.com/api/articles")
+      .then((response) => {
+        return response.json();
+      })
```

Comment:
I'd encourage you to put this into a util reusable function, like I showed in that end of day seminar last week. That way you can then re-use this function over and over again elsewhere.

Also, it's entirely your choice but you may want to use `axios` as it'll help you handle errors easier later on.

Reply:
done